docs/usage.md

```diff
@@ -38,4 +38,12 @@ With MITRE ATT&CK&reg; Enrichment Framework, you can configure and schedule data
     - Click a *Tactic* to view more information about it
     - Under the tab **Techniques** you can check the techniques that the selected tactic uses
 
+7. Open an alert or an incident containing a technique ID
+    - Under **Correlations** check correlated *Tactics*, *Technique*, *sub-techniques*, and other information
+        - For SOAR Framework `v2.1.1` and earlier, check the *Mitigations* tab
+        - For SOAR Framework `v2.2.0` and later, check the *Recommended ATT&CK Mitigations* tab
+
```

Comment:
- For SOAR Framework `v2.1.1` and earlier, check *Mitigations* under **Correlations** tab
- For SOAR Framework `v2.2.0` and later, check *Mitigations* under *Recommended ATT&CK Mitigations* tab